superset/assets/src/explore/components/controls/SubscriberLayer.jsx

```diff
@@ -221,10 +233,28 @@ export default class SubscriberLayer extends React.PureComponent {
     this.forceUpdate();
   }
 
+  handleTitleInclusion(actionType, index) {
+    let subscriptionList = this.state.subscriptionList;
+    subscriptionList[index]['actionType'] = actionType;
+
+    let subscribe_columns = this.state.subscribe_columns;
+    subscribe_columns.length <= index ? subscribe_columns.push({}) : subscribe_columns;
+    subscribe_columns[index]['actions'] = actionType;
```

Comment:
<= means ?

Reply:
As discussed, it was misinterpretation of line.

Reply:
A tarnary expression without a LHS is always confusing. `a=  b ? c : d` 
If we are updating a reference we can use simple if statment here
`if(subscribe_columns.length <= index)  subscribe_columns.push({})`